qiskit/tools/visualization/_state_visualization.py

```diff
@@ -99,57 +127,77 @@ def plot_hinton(rho, title='', filename=None, show=False):
         rect = plt.Rectangle([x - size / 2, y - size / 2], size, size,
                              facecolor=color, edgecolor=color)
         ax2.add_patch(rect)
-
-    ax2.set_xticks(np.arange(0.5, lx+0.5, 1))
-    ax2.set_yticks(np.arange(0.5, ly+0.5, 1))
-    ax2.set_yticklabels(row_names, fontsize=12)
-    ax2.set_xticklabels(column_names, fontsize=12, rotation=-90)
+    if np.any(dataimag != 0):
+        ax2.set_xticks(np.arange(0, lx+0.5, 1))
+        ax2.set_yticks(np.arange(0, ly+0.5, 1))
+        ax2.set_yticklabels(row_names, fontsize=14)
+        ax2.set_xticklabels(column_names, fontsize=14, rotation=90)
     ax2.autoscale_view()
     ax2.invert_yaxis()
-    ax2.set_title('Imag[rho]')
+    ax2.set_title('Imag[rho]', fontsize=14)
     if title:
         plt.title(title)
```

Comment:
This is an existing bug, but when I called:

plot_hinton(rho, title='My Hinton') this sets the title over ax2 instead of Imag[rho]